lib/services/fetch_basket.rb

```diff
@@ -0,0 +1,16 @@
+module Store
+  class FetchBasket
+    def call
+      BASKET.map do |item|
+      {
+        product: item.fetch_product,
+        id: item.id,
+        quantity: item.quantity,
+        price_with_vat: item.price_with_vat,
+        total_price: item.total_price,
+        total_price_with_vat: item.total_price_with_vat
+      }
```

Comment:
OK, sorry, I missed the fact that you need this for delete for. I agree that it's better to stick with incremental IDs in basket view. Just thought you could omit fetching ID here, but I stand corrected. :)